phpunit.xml

```diff
@@ -21,8 +21,8 @@
         <env name="APP_ENV" value="testing"/>
         <env name="BCRYPT_ROUNDS" value="4"/>
         <env name="CACHE_DRIVER" value="array"/>
-        <!-- <env name="DB_CONNECTION" value="sqlite"/> -->
-        <!-- <env name="DB_DATABASE" value=":memory:"/> -->
+        <env name="DB_CONNECTION" value="sqlite"/>
+        <env name="DB_DATABASE" value=":memory:"/>
```

Comment:
Suggestions:
- [ ] Comment out the `DB_` variables at `env.example.test`
- [ ] Add SQLite to `README.md`, suggestions:
```
- Language: PHP 8.1
- Framework: Laravel 9
- Database: PostgreSQL 13, SQLite 3 (tests only)
```
and
```
Make sure you have PHP, PostgreSQL and SQLite installed and running locally. Rename `.env.example.local` to `.env` and change the default falues. You're good to go!
```